lib/report/system.go

```diff
@@ -158,3 +161,27 @@ func fetchEtc(name string) CollectorFunc {
 		)
 	})
 }
+
+// auditLog fetches audit logs from host.
+// The log will contain queries for each of the domains auditlog package is aware of.
+// The resulting file can be post-processed again with:
+//
+//  gunzip -c audit.log.gz | ausearch --interpret --success no ...
+//
+// For example, to see the SELinux denials generated for the gravity domain
+// as a list of allow rules:
+//
+//  gunzip -c audit.log.gz | ausearch --subject gravity_t | audit2allow
+func auditLog() Collector {
+	var subjects []string
+	for _, domain := range auditlog.Domains {
+		subjects = append(subjects,
+			fmt.Sprint("/sbin/ausearch --success no --message all --raw --start yesterday --end today ", "--subject ", domain, " ;"),
```

Comment:
Just curious how this time filter works. Will it collect the entire "yesterday"? Otherwise, if "today" has just started, there may not be enough data to work with.

Reply:
I'll replace `today` with `now` although both seem to mean the same thing. Unless this is not what you meant.